hdtv/database/PGAALibraries.py

```diff
@@ -128,18 +133,18 @@ def open(self):
                 except ValueError:
                     halflife = None
                 gamma = PGAAGamma(
-                    Nuclides(
-                        Z,
-                        A)[0],
+                    Nuclides(Z, A)[0],
                     energy,
                     sigma=sigma,
                     intensity=intensity,
                     halflife=halflife,
-                    k0_comp=self.k0_comp)
+                    k0_comp=self.k0_comp,
+                )
                 self.append(gamma)
         except csv.Error as e:
-            raise HDTVCommandAbort('file %s, line %d: %s' %
-                          (self.csvfile, reader.line_num, e))
+            raise HDTVCommandAbort(
```

Comment:
Codacy found an issue: ['HDTVCommandAbort' may be undefined, or defined from star imports: hdtv.database.common (F405)](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)